### PR TITLE
E2E Tester: Pending Changes

### PR DESCRIPTION
**Pull request by belkind27**
tests adding new entity to app and check if the change added to the pending changes list using jest-pupeteer

### CODE REVIEW

File: packages/amplication-e2e-tester/tests/functions.ts [hunk 1]
```diff
@@ -34,7 +34,8 @@ export async function signUp(
   await (
     await page.waitForXPath(`//button[contains(text(),'${continueBtn}')]`)
   ).click();
-  return page.waitForNavigation();
+  await page.waitForNavigation();
+  return page.waitFor(3000);
```
Comment (iddan) [inline packages/amplication-e2e-tester/tests/functions.ts:38]: Why did you add a wait?

Comment (belkind27) [reply packages/amplication-e2e-tester/tests/functions.ts:38]: sometimes tests using this function randomlly fail because they cant find the xpath of the create new app element, this ensure it doesnt happen

**Review by iddan — APPROVED** [review]
